catalyst-gateway/clippy.toml

```diff
@@ -1,2 +1,3 @@
 allow-unwrap-in-tests = true
 allow-expect-in-tests = true
+arithmetic-side-effects-allowed-binary = [["num_bigint::BigInt", "*"]]
```

Comment:
Could you elaborate why we need exception for such type and such operation ?

Reply:
`BigInt` has arbitrary precision, so as far as I understand it cannot overflow therefore it is safe to exclude this type. This line means that it is OK to use `BigInt` on the left side of any binary arithmetic operation (such as +, -, *, etc.). Now I think that I should use `arithmetic-side-effects-allowed = ["num_bigint::BigInt"]` instead.

Reply:
Yup, it can not overflow.  Agree with this exception.